custom_model.py

```diff
@@ -54,7 +54,19 @@ def __init__(
         self.log_softmax = sb.nnet.activations.Softmax(apply_log=True)
 
     def forward(self, x, hx=None):
-        """List of computations from input to output predictions"""
+        """
+            List of computations from input to output predictions
+
+            hx: tuple(torch.tensor, torch.tensor), optional
+                A tuple containing the initial hidden state and cell state:
+                (h_0, c_0). Each should have shape:
+                (num_layers * num_direction, batch_size, hidden_size).
+                If None, zeros are used.
+            
+            return_hidden: bool
+                If True, returns a tuple (output, hidden), where hidden is a tuple (h_n, c_n)
+                containing the hidden state and cell state for the last time step.
+        """
```

Comment:
Providing a meaningful documentation of the usage of  forward function is critical here to help with maintainability.

Also a validation for the hidden state shape can be useful here